__tests__/api/auth-test.js

```diff
@@ -2,15 +2,17 @@ import axios from 'axios'
 import MockAdapter from 'axios-mock-adapter'
 import {
   getCurrentAuthenticatedUser,
-  authenticateUser,
   ENDPOINTS,
   isUserAuthenticated,
   signOutUser,
+  setAuthToken,
+  getUserMetadata,
 } from '../../src/api/auth'
 import { HTTP_STATUS_CODES } from '../../src/api'
-import { SIGN_IN_RESPONSE } from '../mock_responses/mock-auth-response'
+import { METADATA_RESPONSE } from '../mock_responses/mock-auth-response'
 
-describe('Check authentication', () => {
+describe('Check get user metadata', () => {
+  const MOCK_AUTH_TOKEN = 'A'
```

Comment:
Do we need to care about this warning about hard-coded values?

Reply:
Nope